damnit/backend/extract_data.py

```diff
@@ -106,6 +108,24 @@ def extract_in_subprocess(
     for m in match:
         args.extend(['--match', m])
 
+    if sandbox:
+        bubblewrap = Bubblewrap()
+        with contextlib.suppress(Exception):
```

Comment:
```suggestion
        with contextlib.suppress(FileNotFoundError):
```

Is that what we're trying to catch? Do we expect it to come up in real use, or only in testing?